data-raw/create_dataset/create_dataset.R

```diff
@@ -0,0 +1,52 @@
+library(tm)
+library(stringr)
+library(SnowballC)
+library(parallel)
+library(Matrix)
+library(tidyverse)
```

Comment:
Are all of these dependencies used? I only see `tm` `stringr` and `tidyverse` below.